src/main/java/edu/umich/its/cpm/AttachmentHandler.java

```diff
@@ -0,0 +1,65 @@
+package edu.umich.its.cpm;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.protocol.HttpContext;
+import org.apache.http.util.EntityUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.util.HashMap;
+
+/**
+ * Created by pushyami on 8/26/16.
+ */
+public class AttachmentHandler {
+    private static final Logger log = LoggerFactory.getLogger(AttachmentHandler.class);
+
+    @Autowired
+    private Environment env;
+
+    public Environment getEnv() {
```

Comment:
Yes, but i added those for writing some Junit test. As i could't find an easy way to write test with out getter/setter